wordpress-mobile/AztecEditor-Android
# [Bug] Fix crash and add unit test to cover the issue.

### Fix

Fixes: https://github.com/wordpress-mobile/WordPress-Android/issues/20738

Fix a crash that occurs when a gutenberg post is being edited and there is a block with text color on it, it's possible for the color to be malformed or not parselable. The reasoning for how the Color is empty is not clear, but the function is clear that an empty string is possible. Added unit test to also confirm this covers both an empty and null string color.

### Test

1. Run the test project and ensure you see no crash.
2. Ensure `MarkSpanTest` passes.

### Review
@thomashorta or @antonis 

Make sure strings will be translated:
